include/fc/log/zipkin.hpp

```diff
@@ -55,6 +60,7 @@ class zipkin_config {
 
 private:
    std::unique_ptr<zipkin> zip;
+   static bool received_sighup;
```

Comment:
Don't this should be static. Also it is not thread safe and it needs to be.

Reply:
Updated code.